src/classes/FORM_PermissionsValidator_TEST.cls

```diff
@@ -90,10 +90,11 @@ private with sharing class FORM_PermissionsValidator_TEST {
         System.assert(updatedTemplate.permissionErrors.contains('Account'), 'The '
             + 'permissions error should contain the Account object, which the user does not '
             + 'have access to.'); 
-
+/*      // Should we be checking the widget permissions right now??
```

Comment:
@vmartinov-sforg @dfuller114 Should we be checking the widget permissions here and below? It doesn't look like it is currently set up properly yet. 

cc: @ptewson-sfdo

Reply:
I think our work in progress on the widget front will actually take care of this, or at least set us up to do so. For now, let's not worry about it until a widget has arrived

Reply:
Please be careful with looking for specific english terms in error messages since the word "Account" would likely be something different in other languages. I think alternatively the code could use something like `Account.SObjectType.getDescribe().getLabel()` to retrieve the localized label name for the Account object so the assert works in any language.

Reply:
@KietTruongSFDO Mike speaks the truth - we will need that change else this test will fail in translated orgs! I will suggest a change on that line.

Reply:
I've updated this line and a couple others to use the object/field describe instead of a hardcoded string.

Reply:
Thanks @KietTruongSFDO !